_service-details/secchamps.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Managed Security Champions Programme
+subtitle: Helping you build and improve this
+layout: service
+hero_height: is-small
+backlink: /services
+backname: Value
+---
+
+The industry is pretty unanimous that your organization needs a security champions programme. But where are you going to find time to develop a high value and high impact programme for this, alongside your already full schedule?
+
+That's where Bounce Security comes in.
+
+We use our significant experience in application security as well as our long experience of working directly with developers and within developement organizations to help you build and operate a security champions programme that works for your organization.
+
+We have seen enough Security Champion Programmes in the past to understand well the practices that makes a programme successful and anti-patterns that may reduce the programme's efficiency or even cause it to fail.
+
+With our focus on activities that bring value and impact, the outcome will be interested and engaged developers who are incentivised to excel and provided the resources and support to do so. In turn, this will lead to an improved application security culture within the organization and less security effort required later on in the process.
+
+Our "Managed Security Champions Programme" service is split into a set of options that can be mixed and matched according to what will bring you the highest value right now.

```

Comment:
I think its important to avoid "chinese menu", instead the flexibility should be seen as a benefit. 
E.g., something like `Our "Managed Security Champions Programme" service is based on a flexible package that is customized to your particular needs, based on what will bring you the highest value right now.`

Reply:
I used your wording.